src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -103,7 +110,9 @@
     }
   }
 
-  function MdFabSpeedDialFlingAnimation() {
+  function MdFabSpeedDialFlingAnimation($timeout) {
+    function delayDone(done) { $timeout(done, cssAnimationDuration, false); }
+
     function runAnimation(element) {
```

Comment:
No need... I was simply curious. 

It is my goal that the ngMaterial2 implementatioins should require consistent uses of gesture listeners and  uses of `$animateCss( )`, or `element.style({ ... });`.